public/app/features/scopes/ScopesFacadeScene.ts

```diff
@@ -0,0 +1,58 @@
+import { SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+
+import { scopesFiltersScene } from './instance';
+import { disableScopes, enableScopes, enterScopesReadOnly, exitScopesReadOnly, getSelectedScopes } from './utils';
+
+interface ScopesFacadeState extends SceneObjectState {
+  // A callback that will be executed when new scopes are set
+  handler?: (facade: ScopesFacade) => void;
+
+  // Prevent rendering the selector by default
+  disabled?: boolean;
```

Comment:
I don't see this used anywhere. Do we need it?

Reply:
The idea of this flag is to allow scenes to prevent the rendering of the filters but maintain the ability to read the value etc.

Not very useful tho'. I'll remove it and we can add it if we need it.